requirements.txt

```diff
@@ -1,7 +1,7 @@
-coverage==4.5.1
+coverage>=4.5.1
```

Comment:
I d prefere fixed versions, we wont have bad surprises if a lib upgrades and breaks its API